src/Storages/ExternalStream/Kafka/KafkaSink.h

```diff
@@ -19,26 +19,26 @@ namespace DB
 
 namespace KafkaStream
 {
-/// Partition Chunk's to paritions (i.e. shards) by the partitioning expression.
-class ChunkPartitioner
+/// Shard Chunk's to shards (or partitions in Kafka's term) by the sharding expression.
+class ChunkSharder
```

Comment:
Now we can refactor it out and reuse it for StorageStream as well in the next PR.